examples/graphs/hopcroft_karp_aizu.rs

```diff
@@ -0,0 +1,47 @@
+// verification-helper: PROBLEM https://onlinejudge.u-aizu.ac.jp/problems/GRL_7_A
+
+use proconio::input;
+use programming_team_code_rust::graphs::hopcroft_karp::get_max_matching;
+
+fn main() {
+    input! {
+        lsz: usize,
+        rsz: usize,
+        m: usize,
+        edge_list: [(usize, usize); m],
+    }
+
+    let mut adj = vec![vec![]; lsz];
+    for &(u, v) in edge_list.iter() {
+        adj[u].push(v);
+    }
+
+    let (matching_siz, l_to_r, r_to_l, mvc_l, mvc_r) = get_max_matching(&adj, rsz);
+
+    assert_eq!(
+        matching_siz,
+        l_to_r.iter().filter(|elem| elem.is_some()).count()
+    );
+    assert_eq!(
+        matching_siz,
+        r_to_l.iter().filter(|elem| elem.is_some()).count()
+    );
```

Comment:
floop the two arrays?

Reply:
i prefer functional style; but also floop would leave a local variable `cnt` in the scope

Reply:
what I meant was

```rs
for arr in {r_to_l, l_to_r} {
  assert_eq!(matching_siz, arr...);
}

Reply:
ill update